### PR TITLE
I created affine.py and bench_affine.py. (I am IST I-REF M1 student)

### PR DESCRIPTION
**Pull request by kokhayas**
I created affine.py and bench_affine.py.

### CODE REVIEW

File: affine.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def f(A, B, X):
```
Comment (matsui528) [inline affine.py:4]: `f` is used here https://github.com/media-comp/2022-matmul/blob/main/simple_matmul.py#L4
So can you change the name of the function?

**Review by matsui528 — COMMENTED** [review]
Thanks, great! Just a few comments.

**Comment by matsui528** [comment]
Can you update `README.md` and `bench_ci.yml` as well?

For `bench_ci.yml`, you can write add one line here. 
https://github.com/media-comp/2022-matmul/blob/main/.github/workflows/bench_ci.yml#L17

**Comment by matsui528** [comment]
And can you translate the title of the PR to English?

**Comment by matsui528** [comment]
oh, one more thing, I suggest you write a summary here (I know that many people don't write anything here, but I prefer to write something)
pr

File: affine.py (hunk 1 above)
Comment (kokhayas) [reply affine.py:4]: Thank you. I changed the name of the function to affine for now.

**Pull request merged**